### PR TITLE
moved multicast template to end of config template

### PR DESCRIPTION
**Pull request by webstean**

### CODE REVIEW

File: src/config.c [hunk 1]
```diff
@@ -1071,15 +1067,19 @@ int config_write_template(const char *file, const struct config *cfg)
...
-			 "# httpreq_key\t\tkey.pem\n"
-			 );
-
+			 "#httpreq_ca\t\ttrusted1.pem\n"
+			 "#httpreq_ca\t\ttrusted2.pem\n"
+			 "#httpreq_dns\t\t1.1.1.1\n"
+			 "#httpreq_dns\t\t8.8.8.8\n"
+			 "#httpreq_hostname\tmyserver\n"
+			 "#httpreq_cert\t\tcert.pem\n"
+			 "#httpreq_key\t\tkey.pem\n");
```
Comment (cspiel1) [inline src/config.c:1076]: Placing multicast config at the end of the template is ok. But here is an unrelated change of the httpreq config. At least this should be two commits. Better two PRs.

File: src/config.c [hunk 2]
```diff
@@ -1071,15 +1067,19 @@ int config_write_template(const char *file, const struct config *cfg)
...
+	(void)re_fprintf(f,
+			 "\n# multicast receivers (in priority order)"
+			 "- port number must be even\n"
+                         "#multicast_listener\t\t224.0.2.21:50000\n"
```
Comment (cspiel1) [inline src/config.c:1080]: Tabs instead of spaces. Use tabstop=8!

File: src/config.c [hunk 3]
```diff
@@ -1071,15 +1067,19 @@ int config_write_template(const char *file, const struct config *cfg)
...
+			 "#multicast_listener\t\t224.0.2.21:50002\n"
+			 "\n");
```
Comment (cspiel1) [inline src/config.c:1082]: One "\n" to much, compared to other module config sections.

File: src/config.c (hunk 1 above)
Comment (webstean) [reply src/config.c]: undid change

File: src/config.c (hunk 3 above)
Comment (webstean) [reply src/config.c]: removed last \n

File: src/config.c [hunk 4]
```diff
@@ -1077,9 +1073,12 @@ int config_write_template(const char *file, const struct config *cfg)
 			 "# httpreq_dns\t\t8.8.8.8\n"
 			 "# httpreq_hostname\tmyserver\n"
 			 "# httpreq_cert\t\tcert.pem\n"
-			 "# httpreq_key\t\tkey.pem\n"
-			 );
-
+			 "# httpreq_key\t\tkey.pem\n");
```
Comment (cspiel1) [inline src/config.c:1076]: unrelated change. The ");" was on the next line.

File: src/config.c (hunk 2 above)
Comment (cspiel1) [reply src/config.c:1080]: Line 1080. Still leading spaces instead tabs.

File: src/config.c (hunk 4 above)
Comment (webstean) [reply src/config.c:1076]: Do we really care - that makes it the same as the other sections?

**Comment by cspiel1** [comment]
@alfredh Oh my god. I used the wrong merge button. Better is "Squash+Rebase", I know. Sorry for that!

**Comment by alfredh** [comment]
no worries

i have also done that :)

Alfred
